### ISSUE TITLE
[Mechanics] - screw heads interfere with batteries

### ISSUE THREAD

**Issue by Akustav** [ut-robotics/picr21-team-kobe#75]
Do normal screw heads interfere with batteries that will be pushed into the box.

![image](https://user-images.githubusercontent.com/5435954/143611737-81367883-03ca-4ba3-b1a5-1f5f7635ffe1.png)

**Comment by erkiveevali** [comment]
Yes they probably will, modified those screwholes (and also the ones that hold the NUC itself) to use countersunk screws instead.
